doc/users/plotting/examples/colormap_normalizations_arbitrarynorm.py

```diff
@@ -0,0 +1,62 @@
+"""
```

Comment:
Thanks for the feedback!
Do you have any suggestions where in particular in the example folders? I knew the example folders was preferred but I did it in the doc folder because that is where similar examples (Other normalization examples) were located.

Sure, I will change all the formatting for the docstrings. Even though I read about that formatting I tried to follow the same formatting that was being used in the nearby classes, but I will change that.